graviton/blackbox.py

```diff
@@ -519,12 +520,71 @@ def dryrun_logicsig(
             ),
         )
 
+    @classmethod
+    def dryrun_app_pair_on_sequence(
+        cls,
+        algod: AlgodClient,
+        teal_and_method1: TealAndMaybeMethod,
+        teal_and_method2: TealAndMaybeMethod,
+        inputs: List[Sequence[PyTypes]],
+        abi_argument_types: Optional[List[Optional[abi.ABIType]]] = None,
+        abi_return_type: Optional[abi.ABIType] = None,
+        is_app_create: bool = False,
+        on_complete: OnComplete = OnComplete.NoOpOC,
+        dryrun_accounts: List[DryRunAccountType] = [],
+    ) -> Tuple[Sequence["DryRunInspector"], Sequence["DryRunInspector"]]:
+        return tuple(  # type: ignore
```

Comment:
to the best of my current understand, to fix this type-hints issue would require an upgrade to python 3.11